pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusCommunityDepAnalyzer.java

```diff
@@ -261,7 +261,11 @@ private Path generateQuarkusProject(Predicate<String> artifactIdSelector, String
     }
 
     private String devtoolsJarName() {
-        return "quarkus-product-bom-" + quarkusVersion + ".json";
+        if ((Boolean) getAddOnConfiguration().get("parentOnly")) {
```

Comment:
Let's rename it to "quarkusCoreOnly" or sth like that.

Alternatively, instead of adding a config property, we could check if the `bomArtifactId` for repository generation is `quarkus-bom`

Reply:
> Alternatively, instead of adding a config property, we could check if the bomArtifactId for repository generation is quarkus-bom

`RepositoryGeneration class` which contains `bomArtifactId`  is not accessible to addons
To make it accessible
- have to make change in `RepoManager`
- have to make change in initialization of addons because no addons take `repositoryGenerationData`

Reply:
if `RepositoryGenerationsData` is often needed in `addons` then it  would be good idea to make it proper interface to make `RepositoryGenerationData` class accessible

Reply:
It should be available via `PigContext.get()`

Reply:
Nope. 
https://github.com/project-ncl/bacon/blob/37979855adc4436ce89821024f7875516ab6306c/pig/src/main/java/org/jboss/pnc/bacon/pig/impl/PigContext.java#L73

Its `RepositoryData` not `RepositoryGenerationData`. 
repositorydata  contains `Collection<GAV>`, `Collection<File>` and `Path repoPath` only

Reply:
`PigContext.pigConfiguration.flow.repositoryConfiguration` sth like that

Reply:
Have Used bomartifacid now. Thanks for the lead